CHANGELOG.md

```diff
@@ -107,6 +107,7 @@
 * [ENHANCEMENT] Runtime Config: Allow operator to specify multiple comma-separated yaml files in `-runtime-config.file` that will be merged in left to right order. #2583
 * [ENHANCEMENT] Query sharding: shard binary operations only if it doesn't lead to non-shardable vector selectors in one of the operands. #2696
 * [ENHANCEMENT] Add packaging for both debian based deb file and redhat based rpm file using FPM. #1803
+* [ENHANCEMENT] Query-frontend: improved Prometheus reponse JSON encoding performance. #2450
```

Comment:
Can you move it up, under "main", please?

Reply:
done @ 7e3ae47e6a6787189f206323179e2fc8a421686d